src/tram/static/js/analyze.js

```diff
@@ -75,7 +75,7 @@ function loadSentences(active_sentence_id) {
 function storeSentences(sentences) {
     stored_sentences = {};
     for (var i = 0; i < sentences.length; i++) {
-        sentence = sentences[i];
+        var sentence = sentences[i];
```

Comment:
Not that it matters here, but in general you almost always want `let` in JavaScript rather than `var`.